src/components/ButtonWithIcon/ButtonWithIcon.js

```diff
@@ -10,6 +10,7 @@ type Props = StrokeButtonProps & {
   icon: React$Node,
 };
 
+// Nit: noPadding prop seems unavailable on StrokeButton component. Should we remove it here?
```

Comment:
We do need it actually, it's available since `StrokeButton` is using `ButtonBase` (passed down with the `delegated` props).

Reply:
Yeah could be useful as a reminder!